code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -601,6 +601,33 @@
 	reagent_state = LIQUID
 	color = "#181818"
 
+/datum/reagent/talum_quem
+	name = "Talum-quem"
+	id = "talum_quem"
+	description = " A very carefully tailored hallucinogen, for use of the Talum-Katish."
+	taste_description = "bubblegum"
+	taste_mult = 1.6
+	reagent_state = LIQUID
+	color = "#db2ed8"
+	metabolism = REM * 0.5
+	overdose = REAGENTS_OVERDOSE
+
+datum/reagent/talum_quem/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(alien == IS_DIONA)
+		return
+
+	var/drug_strength = 29
+	if(alien == IS_SKRELL)
+		drug_strength = drug_strength * 0.8
+	else
+		M.adjustToxLoss(17 * removed)
```

Comment:
17 is a bit much. Also, why is this in Chemistry-Reagents-Toxins, it's not a subtype of toxin?

Reply:
Because space drugs are in here and also aren't a subtype of toxin.

Reply:
That'll be corrected at some point soon.

Reply:
Would you want it to be in the Other file, then?

Reply:
Let's go with Medicine.

Reply:
Given it's a hallucinogen and lethal to non-Skrell, that doesn't sound incredibly medicinal.

Reply:
Other is for things like paint and wood pulp, the Medicine file has most of the things that actually do things when ingested.

Reply:
Less than half of the toxins file is children of the actual toxins reagent, as a note.

Reply:
I'll get around to it.

Reply:
I stand corrected, leave it where it is, adjust the toxicity.

Reply:
Alright. It was unintentional anyway- intended to be 7.